### PR TITLE
Delay HTTP200 to the first file chunk received.

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:none
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline sugar pretty please

**Comment by merlin-northern** [comment]
it works when tested manually but some tests failed, I am on it.

**Comment by merlin-northern** [comment]
> it works when tested manually but some tests failed, I am on it.

ok I know headers I missing.

**Comment by merlin-northern** [comment]
@mender-test-bot  start pipeline sugar pretty -ease

File: api/http/management_filetransfer.go [hunk 1]
```diff
@@ -361,7 +362,7 @@ func (h ManagementController) downloadFileResponse(c *gin.Context, params *fileT
 func (h ManagementController) downloadFileResponseProcessMessage(c *gin.Context,
 	params *fileTransferParams, request *model.DownloadFileRequest, wsMessage *natsio.Msg,
 	deviceTopic string, latestOffset *int64, numberOfChunks *int, responseHeaderSent *bool,
-	ticker *time.Ticker) error {
+	fileInfo *wsft.FileInfo, ticker *time.Ticker) error {
```
Comment (alfrunes) [inline api/http/management_filetransfer.go:365]: This argument looks excessive. Why not just declare the variable on the next line?

**Review by alfrunes — APPROVED** [review]

File: api/http/management_filetransfer.go (hunk 1 above)
Comment (merlin-northern) [reply api/http/management_filetransfer.go:365]: sorry I merged before reading that. I had something in mind while doing that. if I do not find it I will fix with the next patch I do here.

**Pull request merged**